src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -322,36 +318,65 @@ module.exports = class ListObjectsCommandGenerator extends BaseCommandGenerator
 			return;
 		}
 
-		const importedObjects = data.customObjects.filter(
-			customObject => customObject.result.code === IMPORT_0BJECT.SUCCESS
-		);
-		const unImportedObjects = data.customObjects.filter(
-			customObject => customObject.result.code === IMPORT_0BJECT.FAILED
-		);
-
-		if (importedObjects.length) {
-			NodeUtils.println(
-				TranslationService.getMessage(MESSAGES.IMPORTED_OBJECTS),
-				NodeUtils.COLORS.RESULT
-			);
-			importedObjects.forEach(customObject =>
-				NodeUtils.println(
-					`${customObject.type}:${customObject.id}`,
-					NodeUtils.COLORS.RESULT
-				)
-			);
+		this._logImportedObjects(data.successfulImports);
+		this._logUnImportedObjects(data.failedImports);
+	}
+
+	_logImportedObjects(importedObjects) {
+		if (Array.isArray(importedObjects) && importedObjects.length) {
+			NodeUtils.println(TranslationService.getMessage(OUTPUT.IMPORTED_OBJECTS), NodeUtils.COLORS.RESULT);
+			importedObjects.forEach(objectImport => {
+				NodeUtils.println(`    - ${objectImport.customObject.type}:${objectImport.customObject.id}"`, NodeUtils.COLORS.RESULT);
```

Comment:
Also here we can extract the message to a variable to make the line shorter.
Also, could we do tab with tab character \t?